wuvt/static/css/wuvt.css

```diff
@@ -855,50 +838,65 @@ body > footer {
 	width: 100%;
 	text-align: center;
     }
+
     .translateSide {
         transform: translateY(-200%);
         transition: transform 0.4s;
     }
+
     .showMenu {
         transform: translateY(0);
     }
+
     #content {
         margin-left: 0px;
     }
+
```

Comment:
the commit shows a couple dozen lines of extra whitespace added. please don't do this in the future as it makes reviews harder.